packages/framework/ini-files/config-specs.ini

```diff
@@ -2949,6 +2949,98 @@ use RHEL7_POST
 use rhel7_sems-gnu-8.3.0-openmpi-1.10.7-openmp_release-debug_static_no-kokkos-arch_no-asan_no-complex_no-fpic_mpi_no-pt_no-rdc_no-uvm_deprecated-on_no-package-enables
 use PACKAGE-ENABLES|ALL
 
+[rhel8_sems-gnu-8.5.0-openmpi-4.1.6-serial_debug_shared_no-kokkos-arch_no-asan_no-complex_no-fpic_mpi_no-pt_no-rdc_no-uvm_deprecated-on_no-package-enables]
+#uses sems-archive modules
```

Comment:
This line is inaccurate for all of the builds added here, please remove it.  Will approve anyhow, so it can be removed as a separate PR if you feel so inclined (we do need to get this in ASAP, we have two weeks until it has to go live).